package.json

```diff
@@ -27,7 +27,7 @@
     "morgan": "~1.9.1",
     "msgpack5": "^4.2.1",
     "node-jose": "^1.1.3",
-    "nunjucks": "^3.1.7",
+    "nunjucks": "^3.2.1",
```

Comment:
Have you done a brief sanity check on all the web pages with this new version?

Reply:
Yep all seems to be working fine. Unit tests too.